.storybook/webpack.config.js

```diff
@@ -1,131 +1,215 @@
 const path = require('path');
-const createWebpackConfigForDevelopment = require('@commercetools-frontend/mc-scripts/config/create-webpack-config-for-development');
-const createWebpackConfigForProduction = require('@commercetools-frontend/mc-scripts/config/create-webpack-config-for-production');
+const postcssImport = require('postcss-import');
+const postcssPresetEnv = require('postcss-preset-env');
+const postcssReporter = require('postcss-reporter');
+const postcssCustomProperties = require('postcss-custom-properties');
+const postcssCustomMediaQueries = require('postcss-custom-media');
+const postcssPostcssColorModFunction = require('postcss-color-mod-function');
+
+const browserslist = {
+  development: ['chrome', 'firefox'].map(
+    browser => `last 2 ${browser} versions`
+  ),
+  production: ['>1%', 'not op_mini all', 'ie 11'],
+};
 
 const sourceFolders = [
   path.resolve(__dirname),
   path.resolve(__dirname, '../examples'),
   path.resolve(__dirname, '../src'),
 ];
 
-const mcWebpackConfigDev = createWebpackConfigForDevelopment({
-  distPath: '', // not important
-  entryPoint: '', // not important
-  sourceFolders,
-  toggleFlags: {
-    // Disable generation of index.html
-    generateIndexHtml: false,
-  },
-});
-const mcWebpackConfigProd = createWebpackConfigForProduction({
-  distPath: '', // not important
-  entryPoint: '', // not important
-  sourceFolders,
-  toggleFlags: {
-    // Disable extract CSS, as building the production bundles for Storybook
-    // fails with that.
-    enableExtractCss: false,
-    // Disable generation of index.html
-    generateIndexHtml: false,
-  },
-});
-const uikitWebpackConfig = {
-  devtool: 'source-map',
-  module: {
-    rules: [
-      // Storybook uses a plugin to load and render markdown files.
-      {
-        test: /\.md$/,
-        use: [
-          { loader: require.resolve('html-loader') },
-          { loader: require.resolve('markdown-loader') },
-        ],
-      },
-      // For svg icons, we want to get them transformed into React components
-      // when we import them.
-      {
-        test: /\.react\.svg$/,
-        use: [
-          {
-            loader: require.resolve('babel-loader'),
-            options: {
-              babelrc: false,
-              presets: [
-                require.resolve('@commercetools-frontend/babel-preset-mc-app'),
-              ],
-              // This is a feature of `babel-loader` for webpack (not Babel itself).
-              // It enables caching results in ./node_modules/.cache/babel-loader/
-              // directory for faster rebuilds.
-              cacheDirectory: true,
-              highlightCode: true,
-            },
+module.exports = storybookBaseConfig => {
+  storybookBaseConfig.devtool = 'cheap-module-source-map'; // TODO: should we use something differen?
```

Comment:
I've switched the previous setup to use `source-maps`, so that we get nice stack traces. But this makes reloading quite slow of course. Let's give this another try. We can always switch it back later.